workers.rb

```diff
@@ -411,8 +411,10 @@ def detect_statement_of_need(nwo, issue_id)
     end
   end
 
-  def download(issue_id)
-    Open3.capture3("whedon download #{issue_id} #{jid}")
+  def download(issue_id, custom_branch=nil)
+    download_result = Open3.capture3("whedon download #{issue_id} #{jid}")
+    download_result = Open3.capture3("git -C #{jid} checkout #{custom_branch}") if custom_branch
```

Comment:
What does `git -C` do here @xuanxu?

Reply:
@arfon it runs the command on the mentioned directory without changing the path to that dir. So in this case is a single command equivalent to do
```shell
cd #{jid}
git checkout #{custom_branch}
cd ..
```